tests/test_pkcs1.py

```diff
@@ -106,3 +106,35 @@ def test_multiple_signings(self):
         signature2 = pkcs1.sign(message, self.priv, 'SHA-1')
 
         self.assertEqual(signature1, signature2)
+
+    def test_split_hash_sign(self):
+        """Hashing and then signing should match with directly signing the message. """
+
+        message = b'je moeder'
+        print("\tMessage:   %r" % message)
+
+        msg_hash = pkcs1.hash(message, 'SHA-256')
+        print("\tHash: %r" % msg_hash)
+
+        signature1 = pkcs1.sign_hash(msg_hash, self.priv, 'SHA-256')
+        print("\tSignature1: %r" % signature1)
+
+        # Calculate the signature using the original method
```

Comment:
The comment "the original method" implies that the reader knows that your method is something that was added later. This is only true within the context of this pull request, but not later once everything has been merged.